lib/spack/spack/package.py

```diff
@@ -1423,7 +1423,7 @@ def do_patch(self):
             try:
                 with fsys.working_dir(self.stage.source_path):
                     self.patch()
-                tty.debug('Ran patch() for {0}'.format(self.name))
+                tty.msg('Ran patch() for {0}'.format(self.name))
```

Comment:
It *might* be useful to consider integration testing for this message if displaying this during `spack patch` is a requirement from users.

Reply:
Separately, is the verbosity of this message (and/or other spack messages) something that we would like to change across spack commands? I think there may be room for a more general/flexible output stream configuration mechanism to avoid making PRs like this, but I'm not sure if it's justified yet.

Reply:
> > A recent PR
> 
> Could you link the specific one so the commit message has it? git blame says #17546?

Yes, this is https://github.com/spack/spack/commit/605c1a76e08034e93297e0c36c4dc8bf51be0cf6.  It was a change I made before a release in order to significantly curtail the output.

Reply:
> Separately, is the verbosity of this message (and/or other spack messages) something that we would like to change across spack commands? I think there may be room for a more general/flexible output stream configuration mechanism to avoid making PRs like this, but I'm not sure if it's justified yet.

We've discussed supporting different verbosity levels along the lines of Python logging.  The path that was agreed on by the core team (around the time of the verbosity reduction change) was to use `-d`, `-dd`, and `-ddd` for debug levels.

Reply:
@tgamblin  Since the reduction was your requirement, are you okay with this PR?